services/web/server/src/simcore_service_webserver/login/_2fa_api.py

```diff
@@ -141,21 +157,33 @@ async def send_email_code(
     code: str,
     first_name: str,
     product: Product,
+    user_id: UserID | None = None,
 ):
-    email_template_path = await get_template_path(request, "new_2fa_code.jinja2")
-    await send_email_from_template(
-        request,
-        from_=support_email,
-        to=user_email,
-        template=email_template_path,
-        context={
-            "host": request.host,
-            "code": code,
-            "name": first_name,
-            "support_email": support_email,
-            "product": product,
-        },
-    )
+    try:
+        email_template_path = await get_template_path(request, "new_2fa_code.jinja2")
+        await send_email_from_template(
+            request,
+            from_=support_email,
+            to=user_email,
+            template=email_template_path,
+            context={
+                "host": request.host,
+                "code": code,
+                "name": first_name,
+                "support_email": support_email,
+                "product": product,
+            },
+        )
+    except Exception as exc:
```

Comment:
and similarly here, I think a less generic error is better.

Reply:
improved with `TwilioException`